examples/graphene_apps/python_worker/avalon_worker/work_order_processor.py

```diff
@@ -0,0 +1,364 @@
+            return False
+        # Compute work order request hash
+        req_hash = worker_hash.WorkerHash().calculate_request_hash(
+            input_json["params"])
+        # Compare decrypted request hash with expected value
+        if decrypt_req_hash.hex() == req_hash.hex():
+            return True
+        else:
+            return False
+
+# -------------------------------------------------------------------------
+
+    def _create_work_order_response(self, input_json, out_data,
+                                    session_key, session_key_iv):
+        """
+        Creates work order response JSON object.
+
+        Parameters :
+            input_json: JSON work order request
+            out_data: output data dictionary which has data in plain text
+            session_key: Session key of the client which submitted
+                        this work order
+            session_key_iv: iv corresponding to teh session key
+        Returns :
+            JSON RPC response containing result.
+        """
+        output_json = dict()
+        output_json["jsonrpc"] = "2.0"
+        output_json["id"] = input_json["id"]
+        output_json["result"] = dict()
+        output_json["result"]["workloadId"] = \
+            input_json["params"]["workloadId"]
+        output_json["result"]["workOrderId"] = \
+            input_json["params"]["workOrderId"]
+        output_json["result"]["workerId"] = \
+            input_json["params"]["workerId"]
+        output_json["result"]["requesterId"] = \
+            input_json["params"]["requesterId"]
+        workerNonce = ''.join(random.choices(string.ascii_uppercase +
+                                             string.digits, k=16))
+        output_json["result"]["workerNonce"] = \
+            self.encrypt.byte_array_to_base64(
+            workerNonce.encode("UTF-8"))
+        output_json["result"]["outData"] = [out_data]
+
+        # Encrypt outData
+        self.encrypt.encrypt_work_order_data_json(
+            output_json["result"]["outData"],
+            session_key,
+            session_key_iv)
+        # Compute worker signature
+        res_hash = worker_hash.WorkerHash().calculate_response_hash(
+            output_json["result"])
+        res_hash_sign = self.sign.sign_message(res_hash)
+        res_hash_sign_b64 = self.encrypt.byte_array_to_base64(res_hash_sign)
+        output_json["result"]["workerSignature"] = res_hash_sign_b64
+
+        return output_json
+
+# -------------------------------------------------------------------------
+
+    def _get_mrenclave(self):
+        """
+        Get mrenclave value of Graphene-SGX worker enclave.
+
+        Returns :
+            mrenclave value in case of Graphene-SGX worker as hex string.
+            If SGX environment is not present returns empty string.
+        """
+        try:
+            # Read target info
+            with open('/dev/attestation/my_target_info', 'rb') as fd:
```

Comment:
Is hardcoding need of the hour?

Reply:
Not really.  These are pseudo files used by Graphene-SGX for attestation. I can define some   string constant